### PR TITLE
Feat/Achievements Page

### PR DESCRIPTION
**Pull request by Sasikuttan2163**
Work in progress
Depends on https://github.com/dval-in/dvalin-frontend/pull/281

- [x] parse achievement category data
- [x] read achievement data in frontend
- [x] parse each achievement data and organize into categories
- [ ] allow user to mark and unmark achievement completed

### CODE REVIEW

File: src/services/data.service.ts [hunk 1]
```diff
@@ -21,15 +21,24 @@ class DataService {
...
+	async fetchData(
+		type: 'Character' | 'Weapon' | 'AchievementCategory'
+	): Promise<Result<void, Error>> {
 		const isDev = config.DEBUG;
 		try {
 			let files;
 			if (isDev) {
-				const dirPath = join('<path>/dvalin-data/data/EN', type);
+				const dirPath = join(
+					'/run/media/sasikuttan2163/Volume/Dev/Dvalin/dvalin-data/data/EN',
```
Comment (maalni) [inline src/services/data.service.ts:39]: I would move this path into an env var.

Comment (Sasikuttan2163) [reply src/services/data.service.ts:39]: Sorry haha didn't notice

File: .env.example [hunk 2]
```diff
@@ -14,4 +14,5 @@ DATABASE_URL="postgresql://user:pass@localhost:5432/database"
 REDIS_HOSTNAME="localhost"
 REDIS_PORT=6379
 REDIS_PASSWORD=""
-DEBUG="false" # if true will allow the sending of id's
\ No newline at end of file
+DEBUG="false" # if true will allow the sending of id's
+DATA_DIR="<path to>/dvalin-data/data/EN"  # path to use for data directory when debug is enabled
```
Comment (maalni) [inline .env.example:18]: Maybe we should prefix debug vars with DEBUG_

**Review by maalni — DISMISSED** [review]

**Comment by Sasikuttan2163** [comment]
I'll have to start over in the backend

**Pull request closed without merging**